System/EventManager/event_logger.h

```diff
@@ -216,6 +216,7 @@ typedef enum
   EL_CORE_GROUP_EH_RESPOND_WITH_HIGHER_LEVEL_RULE,  //!< EH_Rule でマッチしたが，そのルールで対応せずに，上位のルールで対応させた（詳細は event_handler.h 参照）
   EL_CORE_GROUP_COMMAND_ANALYZE,
   EL_CORE_GROUP_COMMAND_DISPATCHER,
+  EL_CORE_GROUP_CDIS_EXEC_STS,
```

Comment:
この２つ眺めると，ん？ってなるので，命名考えたいね．

```suggestion
  EL_CORE_GROUP_COMMAND_DISPATCHER_INTERNAL_ERR,
  EL_CORE_GROUP_COMMAND_DISPATCHER_EXEC_ERR,
```

とか？

Reply:
```
EL_CORE_GROUP_CDIS_INTERNAL_ERR,
EL_CORE_GROUP_CDIS_EXEC_ERR,
```
はダメですかね？（長いので笑）

Reply:
いいよ～